CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ These changes likely break some apps, please read the
 
 * Removed deprecated types including:
   - dialog.FileIcon (now widget.FileIcon)
+  - widget.Radio (now widget.RadioGroup)
+  - widget.AccordionContainer (now widget.Accordion)
+  - layout.NewFixedGridLayout() (now widget.NewGridWrapLayout())
```

Comment:
Typo in here, it's not `widget.NewGridWrapLayout` but `layout`

Reply:
Good point. I remember thinking that it was weird that we only deprecated widget stuff 😂